src/nodedef.cpp

```diff
@@ -366,7 +366,7 @@ void ContentFeatures::reset()
 	climbable = false;
 	buildable_to = false;
 	floodable = false;
-	rightclickable = true;
+	can_place_onto = true;
```

Comment:
This is misleading - `can_place_onto` is *not* the same as `rightclickable`. Please advise as to what this should be changed to.

Reply:
Never mind, reverted to `rightclickable` for now.